assembl/static2/js/app/components/login/socialMediaLogin.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { form } from 'react-bootstrap';
+
+
+// class SocialMediaItem extends React.Component {
+//   render() {
+//     const typeName = this.props.provider.type;
+//     const name = this.props.provider.name;
+//     const link = this.props.provider.login;
+//     return (
+//       <div className={typeName}>
+//         <a href={link}>{name}</a>
+//       </div>;
+//     )
+//   }
+// }
+
+export class SocialMedia extends React.Component {
+  // TODO: className="`${provider.type}-bg`" on button
+  render() {
```

Comment:
Okay, should have been more clear. TODO: Add CSS to social logins (pair programming with pauline)